data/enraiha0307.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "Akanksha Gahalot",
+  "bio": "**Open Source** | Software Developer | DevOps ",
+  "tags": [
+    "Open Source",
+    "DevOps",
+    "Golang",
+    "Kubernetes",
+    "Docker",
+    "Node.js",
+    "React"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/AkankshaGahalot"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/enraiha0307"
+    },
+    {
+      "icon": "FaHashnode",
```

You're probably be using an invalid icon here. It should be `SiHashnode`

```suggestion
      "icon": "SiHashnode",
```

---

Thanks, I've changed it.